apps/ledger-live-mobile/src/families/polkadot/UnbondFlow/01-Amount.tsx

```diff
@@ -110,6 +112,9 @@ export default function PolkadotUnbondAmount({ navigation, route }: Props) {
   const error = amount.eq(0) || bridgePending ? null : getFirstStatusError(status, "errors");
   const warning = getFirstStatusError(status, "warnings");
   const hasErrors = hasStatusError(status);
+  const isUnbonding = transaction.mode === "unbond";
```

Comment:
Do we need this as we are already in the unbound flow ? The file is in the `UnboundFlow` directory

Reply:
removed ✅